src/components/MainNav/MyProfile/MyProfile.js

```diff
@@ -39,6 +46,7 @@ export default class MyProfile extends Component {
 
     this.state = {
       dropdownOpen: false,
+      userLinks: []
```

Comment:
Please add `,` at the end of the line per consistency.

Reply:
added